MSAL/src/native_auth/public/state_machine/error/RetrieveAccessTokenError.swift

```diff
@@ -65,4 +68,9 @@ public class RetrieveAccessTokenError: MSALNativeAuthError {
     public var isRefreshTokenExpired: Bool {
         return type == .refreshTokenExpired
     }
+    
+    /// Returns `true` if the scope is invalid.
```

Comment:
What if we use something like:
```suggestion
    /// Returns `true` if the scope list is empty.
```